### ISSUE TITLE
improve dataloader usage

### ISSUE THREAD

**Issue by jgcmarins** [jonathangaldino/first-relay-app#9]
I noticed that you've been using loaders like this:
https://github.com/jonathangaldino/first-relay-app/blob/f0d76e576e2cb0f17c2ade2b1d47bd46c05b0399/server/src/modules/item/ItemQuery.js#L33
https://github.com/jonathangaldino/first-relay-app/blob/f0d76e576e2cb0f17c2ade2b1d47bd46c05b0399/server/src/modules/item/ItemQuery.js#L46

Instead of always querying MongoDB, you could use `load` function from dataloader, like this:
https://github.com/FotonTech/golden-stack/blob/d4a72dbc83ad47062e34aa80115eb8317f601e37/packages/server/src/modules/event/EventLoader.ts#L85

https://github.com/FotonTech/golden-stack/blob/d4a72dbc83ad47062e34aa80115eb8317f601e37/packages/server/src/modules/event/EventLoader.ts#L42-L58

**Comment by jonathangaldino** [comment]
Yes! Without a doubt. I didn't add dataloaders before because I was focusing on Relay. I'm already studying dataloader and I will improve on this matter.

**Comment by jonathangaldino** [comment]
I'm using dataloaders now. This technique is op!
